src/stores/attribute-product-create.md

```diff
@@ -19,7 +19,23 @@ _New Attribute Properties_
+1. To determine the type of input control that is used for data entry, set **Input Type** to one of the following:
+
+|Property|Description|
+|--- |--- |
+|Text Field|A single line input field for text.|
+|Text Area|A multiple-line input field for entering paragraphs of text, such as a product description. You can use the WYSIWYG Editor to format the text with HTML tags, or enter the tags directly into the text.|
+|Text Editor|A fully-functioning text editor at the attribute location.|
+|Date|Displays a date value in the [preferred format]({% link stores/attribute-date-time-options.md %}) and [time zone]({% link stores/locale-options.md %}). Date values can be selected from a list or a calendar ( ![Calendar icon]({% link images/images/btn-calendar.png %}) ). <br/><br/>**_Note:_** Depending on your system configuration, _Admin_ users can enter dates directly into a field or select a date from the calendar or list. For information about specifying date and time values, see [Date and Time Custom Options]({% link stores/attribute-date-time-options.md %}).|
+|Yes/No|Displays a drop-down list with pre-defined options of `Yes` and `No`.|
+|Dropdown|Displays a drop-down list of values that accepts only a single selection. The Dropdown input type is a key component of [configurable products]({% link catalog/product-create-configurable.md %}).|
+|Multiple Select|Displays a drop-down list of values that accepts multiple selections.|
+|Price|This input type is used to create price fields that are in addition to the predefined attributes: Price, Special Price, Tier Price, and Cost. The currency used is determined by your system configuration.|
+|Media Image|Associates an additional image with a product, such as a product logo, care instructions, or ingredients from a food label. When you add a media image attribute to the attribute set of a product, it becomes an additional image type, along with Base, Small, and Thumbnail. The media image attribute can be excluded from the [storefront media browser]({% link catalog/media-browser.md %}).|
+|Fixed Product Tax|Lets you define [FPT rates]({% link tax/fixed-product-tax.md %}) based on the requirements of your locale.|
+|Visual Swatch|Displays a swatch that depicts the color, texture, or pattern of a configurable product. A [visual swatch]({% link catalog/swatches.md %}) can be filled with a hexadecimal color value, or display an uploaded image that represents the color, material, texture, or pattern of the option.|
+|Text Swatch|A text-based representation of a configurable product option that is frequently used for size. [Text swatches]({% link catalog/swatches.md %}) can also include hexadecimal color values.|
+|<span class="ee-only">Page Builder</span>|A fully-functioning [Page Builder]({% link cms/page-builder.md %}) workspace at the attribute location that makes it easy to add engaging content to the product page.|
```

Comment:
Needs to be nested or the numbering will be broken

```suggestion
   |Property|Description|
   |--- |--- |
   |Text Field|A single line input field for text.|
   |Text Area|A multiple-line input field for entering paragraphs of text, such as a product description. You can use the WYSIWYG Editor to format the text with HTML tags, or enter the tags directly into the text.|
   |Text Editor|A fully-functioning text editor at the attribute location.|
   |Date|Displays a date value in the [preferred format]({% link stores/attribute-date-time-options.md %}) and [time zone]({% link stores/locale-options.md %}). Date values can be selected from a list or a calendar ( ![Calendar icon]({% link images/images/btn-calendar.png %}) ). <br/><br/>**_Note:_** Depending on your system configuration, _Admin_ users can enter dates directly into a field or select a date from the calendar or list. For information about specifying date and time values, see [Date and Time Custom Options]({% link stores/attribute-date-time-options.md %}).|
   |Yes/No|Displays a drop-down list with pre-defined options of `Yes` and `No`.|
   |Dropdown|Displays a drop-down list of values that accepts only a single selection. The Dropdown input type is a key component of [configurable products]({% link catalog/product-create-configurable.md %}).|
   |Multiple Select|Displays a drop-down list of values that accepts multiple selections.|
   |Price|This input type is used to create price fields that are in addition to the predefined attributes: Price, Special Price, Tier Price, and Cost. The currency used is determined by your system configuration.|
   |Media Image|Associates an additional image with a product, such as a product logo, care instructions, or ingredients from a food label. When you add a media image attribute to the attribute set of a product, it becomes an additional image type, along with Base, Small, and Thumbnail. The media image attribute can be excluded from the [storefront media browser]({% link catalog/media-browser.md %}).|
   |Fixed Product Tax|Lets you define [FPT rates]({% link tax/fixed-product-tax.md %}) based on the requirements of your locale.|
   |Visual Swatch|Displays a swatch that depicts the color, texture, or pattern of a configurable product. A [visual swatch]({% link catalog/swatches.md %}) can be filled with a hexadecimal color value, or display an uploaded image that represents the color, material, texture, or pattern of the option.|
   |Text Swatch|A text-based representation of a configurable product option that is frequently used for size. [Text swatches]({% link catalog/swatches.md %}) can also include hexadecimal color values.|
   |<span class="ee-only">Page Builder</span>|A fully-functioning [Page Builder]({% link cms/page-builder.md %}) workspace at the attribute location that makes it easy to add engaging content to the product page.|
```

Reply:
Done @leslietilling , thank you!
Could you please approve?☀️